src/components/Follow.test.tsx

```diff
@@ -92,31 +88,111 @@ test("find a user by nprofile", async () => {
   screen.getByDisplayValue(npub);
 });
 
-test("find a user by nip-05 identifier", async () => {
-  const bobsIdentifier = "bob@bobsdomain.com";
-  nip05.useFetchImplementation(async () =>
-    Promise.resolve({
-      json: () => Promise.resolve({ names: { bob: BOB_PUBLIC_KEY } }),
-    })
-  );
+const bobsNip05Event = finalizeEventWithoutWasm(
+  {
+    kind: 0,
+    tags: [],
+    content: JSON.stringify({ nip05: bobsNip05Identifier }),
+    created_at: newTimestamp(),
+  },
+  BOB.privateKey
+);
 
-  const [alice] = setup([ALICE]);
-  renderWithTestData(<Follow />, {
-    ...alice(),
-    initialRoute: "follow user",
+/* eslint-disable functional/no-let */
+let shouldMock = false;
```

Comment:
If you define them locally instead of globally, you don't need to overwrite them in beforeAll()